clients/python/agentic-sandbox-client/k8s_agent_sandbox/connector.py

```
@@ -325,18 +446,54 @@ def _connection_strategy(self):
+
     def connect(self) -> str:
         return self.strategy.connect()
 
     def close(self):
         self._pod_ip_resolved = False
         self._pod_ip = None
+        if self._grpc_channel is not None:
+            try:
+                self._grpc_channel.close()
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)